### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by JoshuaSkootsky**
Redux Store updates on thunk with contents of the API call

### CODE REVIEW

File: client/store/boat.spec.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+describe('thunk creators', () => {
+  let store;
+  let mockAxios;
+
+  const initialState = { user: {} };
+
+  beforeEach(() => {
+    mockAxios = new MockAdapter(axios);
+    store = mockStore(initialState);
```
Comment (natiwhitney) [inline client/store/boat.spec.js]: Nice!

File: client/store/boat.spec.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          cost: 140193,
+        },
+        {
+          name: 'High Maintenance',
+          imageUrl: '',
+          description: '',
+          cost: 134655,
+        },
+        {
+          name: 'Aquaholic',
```
Comment (natiwhitney) [inline client/store/boat.spec.js]: These boat names are very funny

File: client/store/boat.spec.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        },
+        {
+          name: 'Squid Pro Quo',
+          imageUrl: '',
+          description: '',
+          cost: 840726,
+        },
+      ];
+      mockAxios.onGet('/api/boats').replyOnce(200, fakeBoats);
+      await store.dispatch(fetchBoats(fakeBoats));
```
Comment (natiwhitney) [inline client/store/boat.spec.js]: @JoshuaSkootsky I am confused ... how can fetchBoats accept an argument if it is a thunk?

Comment (JoshuaSkootsky) [reply client/store/boat.spec.js]: yes, it doesn't do anything with that argument - the test passes without that parameter.

File: client/store/boat.spec.js [hunk 4]
```diff
@@ -91,7 +91,7 @@ describe('thunk creators', () => {
         },
       ];
       mockAxios.onGet('/api/boats').replyOnce(200, fakeBoats);
-      await store.dispatch(fetchBoats(fakeBoats));
+      await store.dispatch(fetchBoats());
```
Comment (JoshuaSkootsky) [inline client/store/boat.spec.js]: @natiwhitney - took the extra param out

**Pull request merged**